packages/stylint-config/example/src/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Lets keep this examples outside of the packages and at root.

Reply:
@ankurkakroo2  My thought to keep it at package specific only,

Reply:
Sure. Lets talk about it. I suggested this because you used both configs in your example.
Also, if we move it out, we can demonstrate everything in 1 example (considering only react )

Reply:
@ankurkakroo2  As discussed move to package folder